### ISSUE TITLE
Reproducing Results

### ISSUE THREAD

**Issue by seansegal** [carla-simulator/reinforcement-learning#13]
Hi,

We have been working to reproduce the results of this repository by training an A2C agent from scratch in the CARLA simulator. You can see our code here: https://github.com/seansegal/carla-rl. 

While we do see a considerable increase in mean episode reward over the course of our training, we have not be able to match the performance in the benchmarks that was reported in the original CARLA paper. We matched all the hyperparameters given in the paper and for those that are not given, we have tried to infer those from the `agent/trained_model/args.txt` file inside this repository. 

Would it be possible to release the training code for this agent? Given that this is part of an open-source project, is there a reason that it has not been released yet?

If releasing the code is not possible, could you please provide some of the following training details:
- Which tasks were used for training (no turns, one turn, routing, dynamic objects)? Were they sampled randomly? Was curriculum learning used?
- Was any reward normalization technique used during training?
- How do you handle this bug (https://github.com/carla-simulator/carla/issues/13) in your training? We have an issue where a car can get stuck in an episode against a static object since the collision is not registered. The experiment only terminates during the timeout. 
- Were the agent's trained with 10 million environment steps or 100 million environment steps (10 million steps for each of the 10 environments)?

Thanks for your help!
Sean, Sergio and Seung-Eun (University of Toronto)

**Comment by eds89** [comment]
Any progress in this?
